react-apps/self-care-portal/self-care-portal-frontend/accelerator/src/detailedAgreementPage/KeyDatesInfo.jsx

```diff
@@ -127,7 +127,7 @@ export const KeyDatesInfo = ({consent, infoLabels, consentType}) => {
     return (
         <div className="keyDatesBody">
             <h5>{keyDatesConfig.keyDatesInfoLabel}</h5>
-            {keyDatesMap}
+            <div className="two-columns">{keyDatesMap}</div>
```

Comment:
Without manually defining styles, can't we use existing bootstrap breakpoints[1] for this?

[1] https://getbootstrap.com/docs/5.3/layout/breakpoints/#available-breakpoints